test/unit/services/matchers/giovanni/model1/matcher.test.js

```diff
@@ -0,0 +1,60 @@
+const MatcherResult = require("../../../../../../app/services/matches-result");
+const Matcher = require("../../../../../../app/services/matchers/giovanni/model1/matcher");
+const model = require("../../../../test-helpers/giovanni/model1/data-model");
+
+describe("matchesGiovanni", () => {
+  test("returns generic error for empty json", () => {
+    const packingListJson = {};
+    const filename = "packinglist.xlsx";
+    const result = Matcher.matches(packingListJson, filename);
+    expect(result).toBe(MatcherResult.GENERIC_ERROR);
+  });
+  test("returns Correct", () => {
+    const filename = "PackingList.xlsx";
+
+    const result = Matcher.matches(model.validModel, filename);
+
+    expect(result).toBe(MatcherResult.CORRECT);
+  });
+  test("returns wrong establishment number for missing establishment number", () => {
+    const packingListJson = {
```

Comment:
you could put all json used here in data-model.js as well

Reply:
Just seen this, I'll do it now. thanks David